Octokit.Tests.Integration/Clients/IssuesClientTests.cs

```diff
@@ -233,6 +242,22 @@ public IssuesClientTests()
     }
 
     [IntegrationTest]
+    public async Task CanRetrieveIssueWithMultipleAssignees()
+    {
+        var issue = await _issuesClient.Get("octokit", "octokit.net", 1171);
+
+        Assert.True(issue.Assignees.Count == 2);
```

Comment:
:lipstick: `Assert.Equal(2, issue.Assignees.Count)` to get the value if this test fails